commands/welcome/help.js

```diff
@@ -12,24 +12,24 @@ module.exports = {
       const command = await client.commands.get(args[0]);
 
       if (!command) {
-        return message.channel.send("Unknown Command: " + args[0]);
+        return message.channel.send(`Unknown Command: ${args[0]}`);
       }
 
-      let embed = new MessageEmbed()
+      const embed = new MessageEmbed()
         .setAuthor(command.name, client.user.displayAvatarURL())
         .addField("Description", command.description || "Description is not provided join support server for help")
-        .addField("Usage", "`" + command.usage + "`" || "Not Provied Join Support server for help")
-        .addField("Aliases", "```" + command.aliases + "```"|| "There is no aliases")
+        .addField("Usage", `\`${command.usage}\`` || "Not Provided Join Support server for help")
+        .addField("Aliases", `\`\`\`${command.aliases}\`\`\``|| "There is no aliases")
         .setThumbnail(client.user.displayAvatarURL())
         .setColor("#000000")
         .setFooter(`Join our support server `);
 
       return message.channel.send(embed).then(m => m.delete({ timeout: 8000 }));
-    } else {
-      const commands = await client.commands;
+    }
```

Comment:
missing `else` function

Reply:
`else` is not a function. It is also not mandatory :), removes unnecessary code!